src/wallet/walletdb.cpp

```diff
@@ -5,6 +5,10 @@
 
 #include <wallet/walletdb.h>
 
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
+
 #include <common/system.h>
```

Comment:
Isn't `bitcoin-config.h` been indirectly included by this `system.h` include? (which btw, doesn't seems to be needed).

Reply:
> Isn't `bitcoin-config.h` indirectly included by this `system.h` include?

It is.

> While this header might be included indirectly elsewhere, including it explicitly makes the build process more robust.

Reply:
it would be good to remove the unused include within this change. walletdb.cpp does not call to any system.h` function.